src/unix/getaddrinfo.c

```diff
@@ -200,3 +200,22 @@ void uv_freeaddrinfo(struct addrinfo* ai) {
   if (ai)
     freeaddrinfo(ai);
 }
+
+
+int uv_if_indextoname(unsigned int ifindex, char* ifname, size_t *sizep) {
+  char ifname_buf[UV_IF_NAMESIZE];
+  if (NULL == if_indextoname(ifindex, ifname_buf)) {
```

Comment:
shouldn't we set `ifname_buf[UV_IF_NAMESIZE - 1]` to zero to be sure is NULL terminated?